utilities/create_site.sh

```diff
@@ -30,4 +30,17 @@ git add index.html
 echo "<html><head><script>window.location.replace('/google-cloud-java/${SITE_VERSION_BASE}/apidocs' + location.search)</script></head><body></body></html>" > apidocs/index.html
 git add apidocs/index.html
 
+cd ..
+mvn site:stage --quiet -Djava.util.logging.config.file=logging.properties -DtopSiteURL=http://googlecloudplatform.github.io/google-cloud-java/site/latest/
+cd tmp_gh-pages
+rm -rf latest
+mkdir latest
+for dir in ../target/staging/*
```

Comment:
I'm confused by this.

It seems like this code is attempting to repeat the regular site creation process one more time, with only difference: it does not update global tmp_gh-pages/index.html to point to latest (it keeps pointing to the actual newest version, which seems reasonable). If that is the case please strongly consider modifying this code to something like (not sure it it is completely valid bash syntax, but...):

```bash
# [...] (lines 1-17)
version_bases=(${SITE_VERSION_BASE} latest)
for version_base in ${version_bases} 
do
  # All the stuff starting from "mvn site:stage" (line 18) to "git add $SITE_VERSION_BASE" (line 27)
  # but replacing ${SITE_VERSION_BASE} with ${version_base}
done
# [...] (lines 28-31)
```
I believe the following code structure reflects the best what the intent of the change is (and is also shorter, potentially easier to maintain and keep latest in real sync with version_base copy)